### ISSUE TITLE
Midpoint Review - Accessibility Feedback - MHV Medical Records, Filters

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#103881]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** The clear filters button should always be present on the page, rather than displaying conditionally depending on whether or not a user has entered something into the field. In general it's better to have a consistent page structure than to have things appear/disappear conditionally.
- [ ] **Consider:** When sorting allergies, medical conditions, etc, the option to sort "Newest to oldest" may not match users' mental models about their own medical experiences --- eg. an allergy that I've known about since I was 3 and an allergy I've known about since I was 17 may be out of order depending on when they were logged by VA. That may increase cognitive load when trying to understand the results displayed. Consider ways of making it clear that we're sorting by the date added.
- [ ] **Coding note:** I missed the Design System Council meeting on the filter/sort pattern, but my understanding is that sorting with no "sort" button was approved to move forward. I would very much encourage there to be an [`aria-live` region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) that includes some kind of announcement that results have been sorted when the sort action is triggered. Please feel free to reach out to me in Slack if anyone from the team wants to talk about this in more detail.

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
